### PR TITLE
App restructure + Storage disclaimer

### LINKED ISSUE #505: [storage] Display "Beta version" message on buy screen
Alert users that we are in Beta version. This message should only be displayed on the Buy page.

The text should be "DISCLAIMER: Storage services are in **BETA** version and still undergoing final testing before the official release.  IOV Labs do not give any warranties regarding the suitability or usability of this module and the related technologies. Decentralized storage is still considered **experimental** technology and any purchasing of storage space should be done at the user’s own risk."

image

### PR DESCRIPTION
**Pull request by farinalvaro**
- Moves most of the logic from `AppComponent` to `MarketComponent` (where we can read from `AppContext`).
- Extracts AlertInfo to new component and moves messages logic to AppContext
- Adds disclaimer message - Closes #505

### CODE REVIEW

File: src/constants/strings.ts [hunk 1]
```diff
@@ -2,4 +2,12 @@ export const ZERO_ADDRESS = '0x'.padEnd(42, '0')
 
 export const ZERO_BYTES = '0x'.padEnd(64, '0')
 
+export const STORAGE_DISCLAIMER_MSG = `DISCLAIMER: Storage services are in BETA version 
+and still undergoing final testing before the official release.
+ IOV Labs do not give any warranties regarding the suitability
```
Comment (julianmrodri) [inline src/constants/strings.ts:7]:
```suggestion
 IOV Labs does not give any warranties regarding the suitability 
```

**Review by julianmrodri — DISMISSED** [review]
Love it! Fixed a typo in the original mesage (my bad). Besides that LGTM!

File: src/constants/strings.ts (hunk 1 above)
Comment (julianmrodri) [reply src/constants/strings.ts:7]: We should include this fix. It was wrong in the original message sorry.

Comment (farinalvaro) [reply src/constants/strings.ts]: 👍 done

**Review by julianmrodri — APPROVED** [review]
LGTM

**Pull request merged**